proposed/0007-Address_relocation.md

```diff
@@ -0,0 +1,40 @@
+- Feature Name: Prevent any single address being used more than once during address relocation.
+- Type enhancement
+- Related components routing, safe_vault
```

Comment:
according to the description below, it seems the implementation can be achieved solely in routing lib.
As I understand, this address relocation shall also be done solely by routing lib (unless it also covers client node, which client lib may need to be involved to provide account info to calculate connection address).
So, I think vault shall be removed as a related component.